packages/fxa-settings/src/components/BentoMenu/en-US.ftl

```diff
@@ -0,0 +1,12 @@
+# BentoMenu component
+
+bento-menu-title = Firefox Bento Menu
+bento-menu-firefox-title = Firefox is tech that fights for your online privacy.
+
+bento-menu-vpn = { -product-mozilla-vpn }
+bento-menu-monitor = { -product-firefox-monitor }
+bento-menu-pocket = { -product-pocket }
+bento-menu-firefox-desktop = Firefox Browser for Desktop
+bento-menu-firefox-mobile = Firefox Browser for Mobile
```

Comment:
Hmm. These strings on lines 9 and 10 look weird to me. I can't recall another spot where mozilla websites use the word "browser" in describing firefox or fenix, but idk. 

Aside from whether these are the right strings, I'm not sure if these should be localized like "Firefox Accounts" is, where "Firefox" is treated as a brand, but "Accounts" can be translated:

```
# “Accounts” can be localized, “Firefox” must be treated as a brand.
-product-firefox-accounts = Firefox Accounts
```

or if we just want to leave both as untranslated product/brand strings, and call it a day.

@flodolo or @peiying2, do you have any guidance on the right strings to use for desktop and mobile firefox in this "bento menu"? I'm not sure if mobile here means just android or android/ios.